DiceKit/AdditionExpression.swift

```diff
@@ -72,3 +72,27 @@ public func + <R: protocol<ExpressionType, Equatable>>(lhs: Int, rhs: R) -> Addi
 public func + <L: protocol<ExpressionType, Equatable>>(lhs: L, rhs: Int) -> AdditionExpression<L, Constant> {
     return AdditionExpression(lhs, Constant(rhs))
 }
+
+// MARK: - CustomDebugStringConvertible
```

Comment:
I think we should list `CustomStringConvertible` before `CustomDebugStringConvertible`, and both of them right before the `Equatable` section.